src/transformation/wavefront.rs

```diff
@@ -0,0 +1,42 @@
+use crate::shape::TriMesh;
+
+#[cfg(feature = "wavefront")]
+use obj::*;
+#[cfg(feature = "wavefront")]
+use std::path::PathBuf;
+
+#[cfg(feature = "wavefront")]
```

Comment:
This are unnecessary because the `pub mod wavefront;` itself is already feature-gated.
```suggestion
use obj::*;
use std::path::PathBuf;
```